src/DynamoCoreWpf/Views/PackageManager/Controls/NumericUpDownControl.xaml.cs

```diff
@@ -43,24 +43,25 @@ public string Watermark
                 new FrameworkPropertyMetadata("0"));
 
         // The Value of the numerical up/down control. Bind to this property
-        public int Value
+        public string Value
```

Comment:
what is this used for?

Reply:
@dnenov when publishing a new version where is this string converted to a number? At least I am assuming that a number is needed in the request for the version information?

Reply:
The properties of the PublishPackageViewModel, and they are also strings. https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs#L294 It gets used in the BuildPackage https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs#L1646, so it's strings all the way really. But there is an int check built inside those properties.